contracts/src/Summa.sol

```diff
@@ -77,6 +90,7 @@ contract Summa is Ownable {
             );
             uint256 index = ownershipProofByAddress[addressHash];
             require(index == 0, "Address already verified");
+            //Offsetting the index by 1 to distinguish with the case when the proof hasn't been submitted (the storage slot would be zero)
```

Comment:
Let's illustrate with an example to clarify the behavior of the `submitProofOfAddressOwnership` function. 

In the first transaction, if two proofs are submitted, the `ownershipProofByAddress` mapping will be updated as:
```
{
  hash1: 1,
  hash2: 2
}
```
In a subsequent transaction with three proofs, the mapping will be updated to:
```
{
  hash1: 1,
  hash2: 2,
  hash3: 1,
  hash4: 2,
  hash5: 3
}
```
Is this how the function is supposed to work?

Reply:
Fixed.